client/src/modules/depots/modals/depot.modal.html

```diff
@@ -0,0 +1,48 @@
+<form name="DepotForm" bh-submit="DepotModalCtrl.submit(DepotForm)" novalidate>
+  <div class="modal-header">
+  <ol class="headercrumb">
+    <li ng-if="DepotModalCtrl.isCreating" class="title">
+      <span translate>DEPOT.ADD_DEPOT</span>
+      <label class="badge badge-warning" translate>FORM.LABELS.CREATE</label>
+    </li>
+    <li ng-if="!DepotModalCtrl.isCreating" class="title">
+      <span translate>DEPOT.EDIT_DEPOT</span>
+      <label class="badge badge-warning" translate>FORM.LABELS.UPDATE</label>
+    </li>
+  </ol>
+</div>
+
+  <div class="modal-body">
+
+    <div class="form-group" ng-class="{ 'has-error' : DepotForm.$submitted && DepotForm.text.$invalid }">
+      <label class="control-label" translate>DEPOT.LABEL</label>
+      <input name="text" ng-model="DepotModalCtrl.depot.text" autocomplete="off" class="form-control" required>
+      <div class="help-block" ng-messages="DepotForm.text.$error" ng-show="DepotForm.$submitted">
+        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+      </div>
+    </div>
+
+    <div class="form-group">
```

Comment:
This HTML should follow [Bootstrap's standard markup for checkboxes and radiobuttons](https://getbootstrap.com/docs/3.3/css/#checkboxes-and-radios).  It should look something like this:
```html
<div class="checkbox">
  <label>
    <input type="checkbox" .... > 
    <span translate>SOME.KEY</span>
   </label>
</div>
```
Since there is no validation, you don't need the "form-group" styles.